I/ImageMagick7/ImageMagick/build_tarballs.jl

```diff
@@ -0,0 +1,62 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder
+name = "ImageMagick"
+upstream_version = v"7.1.0-60"
+version = VersionNumber(upstream_version.major, upstream_version.minor, upstream_version.patch)
+
+# Collection of sources required to build imagemagick
+sources = [
+    GitSource("https://github.com/ImageMagick/ImageMagick",
+              "212bed9bab497c0f738c0f11ded1f34656ff154e"),
+    DirectorySource("./bundled"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/ImageMagick*/
+atomic_patch -p1 ../patches/check-have-clock-realtime.patch
+if [[ "${target}" == *-mingw* ]]; then
+    export LDFLAGS="-lurlmon"
```

Comment:
Uhm, this means everything is being linked to liburlmon, which I'm not sure it was the original intent. I'll try to have a look when I have some time

Reply:
Note that `coders/url.c` specifically does seem to have the right flag enabled: https://github.com/ImageMagick/ImageMagick/blob/6558dbbd880dc1caa49c9f7d319d668e61c2dcab/coders/Makefile.am#L1393 I think the issue might be that this flag then isn't used when linking MagickCore

Reply:
I left a comment in https://github.com/ImageMagick/ImageMagick/pull/5727#discussion_r1096556972, perhaps I'll open an issue at some point.  I added a temporary patch, but I'm not really sure how this is supposed to work.